app/data/projects.json

```diff
@@ -1113,5 +1113,12 @@
     "projectRepository": "https://github.com/Tendrl",
     "projectWebsite": "http://tendrl.org/",
     "category": "Storage"
+  },
+  {
```

Comment:
Would you mind adding this in as close to alphabetically as possible - it just makes the JSON file easier to manually search when necessary.

Reply:
That was my original intent, and then it looked like the file was not in alphabetical order so I got the impression that the sorting was happening external to the file. I will fix.